man/fi_gni.7.md

```diff
@@ -182,13 +189,23 @@ For *FI_GNI_EP_OPS_1*, the currently supported values are:
 *GNI_HASH_TAG_IMPL*
 : Use a hashlist for the tag list implementation.  The value is of type uint32_t.
 
+The `native_amo` function allows the user to call GNI native atomics
+that are not implemented in the libfabric API.
+The parameters for native_amo are the same as the fi_atomic function
+but adds the following parameter:
+
+*enum gnix_fab_req_type req_type*
+: The req_type's supported with this call are GNIX_FAB_RQ_NAMO_AX, and
+GNIX_FAB_RQ_NAMO_AX_S.
+
```

Comment:
Can you add a short description of the req_type values in parens after each one?  e.g., "(AND and XOR)".